public/js/build/main.js

```diff
@@ -0,0 +1,96 @@
+var app = angular.module("app", ["ngRoute"]);
+	app.config(['$interpolateProvider','$routeProvider', function ($interpolateProvider, $routeProvider){
+		$interpolateProvider.startSymbol('{[{');
+		$interpolateProvider.endSymbol('}]}');
+
+		$routeProvider.when("/",{
+	        templateUrl: "views/dashboard.html",
+	        controller: "courseGenerator"
+	    }).when("/dashboard",{
+	        templateUrl: "views/dashboard.html",
+	        controller: "courseGenerator"
+	    }).when("/addRubric",{
+	        templateUrl: "views/rubric.html",
+	        controller: "courseGenerator"
+	    }).otherwise({
+	        redirectTo: "/"
+	    })
+
+	}]);
+
+
+
+	app.controller('homeController', ['$scope', '$http', '$routeParams','$location', function($scope, $http, $routeParams, $location){
```

Comment:
If a controller/section of code provides no functionality, ensure it is removed.